_content/essays/jhanas-and-the-lay-disciple_bodhi.md

```diff
@@ -11,7 +11,10 @@ tags:
   - theravada
   - jhana
   - stages
-pages: "14"
+  - lay
+year: 2004
```

Comment:
Yeah, we just use the date we got. What else can we do? If evidence comes along that it was published earlier, we can always update the value at that time 😊